Pdk

raphink opened the pull request.

raphink committed:
Update .sync.yml for travis

raphink committed:
Update dependencies and supported OSes

raphink committed:
Rubocop autocorrect

raphink committed:
Rubocop: Lint/ParenthesesAsGroupedExpression

raphink committed:
Fix Style/RegexpLiteral

raphink committed:
Rubocop lint

raphink committed:
Remove rspec opts

raphink committed:
Use beaker still

raphink committed:
Beaker

raphink committed:
Adapt spec_helper_acceptance

abraham1901 commented:
Good job!

raphink committed:
pdk update

raphink committed:
Switch to litmus

Signed-off-by: Raphaël Pinson <raphael.pinson@camptocamp.com>

raphink committed:
Add fixtures for litmus

raphink committed:
Adapt spec_helper_acceptance

Signed-off-by: Raphaël Pinson <raphael.pinson@camptocamp.com>

raphink committed:
Turn off rubocop for now

DavidS commented:
added a comment with a fix to https://github.com/puppetlabs/puppet_litmus/issues/317#issuecomment-766739435

raphink committed:
Simplify spec_helper_acceptance

as per https://github.com/puppetlabs/puppet_litmus/issues/317#issuecomment-766739435

Signed-off-by: Raphaël Pinson <raphael.pinson@camptocamp.com>

raphink committed:
spec_helper_acceptance_local

raphink committed:
use_litmus

Signed-off-by: Raphaël Pinson <raphael.pinson@camptocamp.com>
